docs/plugins/composability.md

```diff
@@ -0,0 +1,585 @@
+---
+id: composability
+title: Composability System Migration
+description:
+  Documentation and migration instructions for new composability APIs.
```

Comment:
```suggestion
# prettier-ignore
description: Documentation and migration instructions for new composability APIs.
```

For SEO, Docusaurus needs it in one line. But prettier wants to split. https://github.com/backstage/backstage/pull/3927